modules/ROOT/pages/design-mocking-service.adoc

```diff
@@ -3,16 +3,21 @@ ifndef::env-site,env-github[]
 include::_attributes.adoc[]
 endif::[]
 
-Use the mocking service to provide a public link to the API you are designing. The mocking service returns the responses (both HTTP status codes and example payloads) that are defined in your API specification, and is valuable for testing or for simply exploring how your API behaves.
+Use the mocking service to provide a public link to the API you are designing. The mocking service returns the responses (both HTTP status codes and example payloads) that are defined in your API specification. The mocking service is valuable for testing or for simply exploring how your API behaves.
 
 == Before You Begin
 
 You must be working in a project to create an API specification in API Designer.
 
 == About this task
+
 Simulating calls to the API is a critical design task that helps you troubleshoot problems and demo the API to prospective users even before you have implemented it. You can simulate calls to the API in API Designer before publishing the API specification or in Exchange after publishing the API specification there.
 
-When you activate the mocking service, it exposes a mock service that you can use for testing in lieu of an actual implementation. If you have already specified a base URI (RAML) or base path (OAS), the mocking service comments it out. When you deactivate the mocking service, it removes the base URI or base path that it used and uncomments your base URI or base path. For example, if the base URI in a RAML API specification is `+http://jsonplaceholder.typicode.com+`, the mocking service replaces it temporarily like this, where `+https://anypoint.mulesoft.com/mocking/api/v1/links/+`, etc. is the URI used by the mocking service:
+When enabled, the mocking service provides a link you can use to test the API. You can share this link with other Anypoint Platform users or with external users who need to see how the API behaves.
+
+When you activate the mocking service, it exposes a mock service that you can use for testing in lieu of an actual implementation. If you have already specified a base URI (RAML) or base path (OAS), the mocking service comments it out. When you deactivate the mocking service, it removes the base URI or base path that it used and uncomments your base URI or base path. 
```

Comment:
Before:
![2020-10-07_09-16-21 (1)](https://user-images.githubusercontent.com/12468128/95329821-f3412880-087d-11eb-8b45-0adeec92f74b.gif)

Now the Specification is not modified:
![2020-10-07_09-18-21 (1)](https://user-images.githubusercontent.com/12468128/95329987-37342d80-087e-11eb-8128-183c77080e9c.gif)